x-pack/plugins/cloud_security_posture/common/types/rules/v4.ts

```diff
@@ -6,15 +6,11 @@
  */
 
 import { schema, TypeOf } from '@kbn/config-schema';
+import { CspBenchmarkRulesStates } from '@kbn/cloud-security-posture-common';
+import { ruleStateAttributes, rulesStates } from '@kbn/cloud-security-posture-common/schema';
```

Comment:
same problem with versioning here. We lose information about the version when importing from our shared package without versioning. Again to make a breaking change you would need to version it somehow on the package level, that's why it's better to move the whole versioned types and schemas to our common package